### ISSUE TITLE
$workspaceRoot variable is misleading

### ISSUE THREAD

**Issue by woigl** [formulahendry/vscode-code-runner#339]
There is the following configuration variable:
`$workspaceRoot: The path of the folder opened in VS Code`

It is actually refering to the parent folder of the extecuted script instead of pointing to the root of the workspace in which the workspace file is residing.

Is there a variable or can there be a variable introduced pointing to the the folder in which the workspace file resides?

**Comment by formulahendry** [comment]
It is really `The path of the folder opened in VS Code`: https://github.com/formulahendry/vscode-code-runner/blob/9ce19bfed2e0c53d4a9da26ae3d4eb5239a1f878/src/codeManager.ts#L352

Could you share more details, so that I could repro the issue from my side?

**Comment by woigl** [comment]
Yes, that is correct. The value of the variable is according to the description, but the name of the variable is misleading. The name "workspaceRoot" would tell me that it points to the root folder of my workspace (I assume the folder that contains my workspace file.

Anyway, is there a possibility that you introduce another variable, which is actually really pointing to the workspace root (the folder that my workspace file is located)?

**Comment by formulahendry** [comment]
Are you working on multi-root workspaces.? Do you mean the `.code-workspace`?

**Comment by woigl** [comment]
Yes, I've added multiple folders to a .code-workspace.
> Are you working on multi-root workspaces.? Do you mean the .code-workspace?

**Comment by formulahendry** [comment]
So, your requirement is: having a variable that points to the folder of `.code-workspace`?

**Comment by woigl** [comment]
Exactly 😄 
> So, your requirement is: having a variable that points to the folder of .code-workspace?
